content/attacks/posts/2017-11-19-Tether.md

```diff
@@ -0,0 +1,33 @@
+---
+date: 2017-11-19
+target-entities: Tether
+entity-types:
+  - Token Issuer
+  - Stablecoin
+attack-types: Wallet Hack
+title: "Tether Faces $31 Million Security Breach"
+---
+
+## Summary
+
+On November 19, 2017, Tether, a company behind the USDT stablecoin pegged 1:1 to the US dollar, [announced a malicious action by an external attacker](https://www.coindesk.com/markets/2017/11/21/tether-claims-30-million-in-us-dollar-token-stolen/). The attacker maliciously removed tokens from the Tether Treasury wallet and sent them to an unauthorized Bitcoin address. As a result, approximately $31 million worth of USDT was taken. In response to the breach, Tether temporarily suspended its back-end wallet service and initiated steps to prevent the stolen coins from entering the ecosystem. New builds of Omni Core, the software used to support Omni Layer transactions, were provided to the community. This move effectively created a temporary hard fork in the Omni Layer, preventing any tokens from being sent from the attacker's address.
```

Comment:
how did they send tether tokens to a bitcoin address?  This doesnt make sense to me

Reply:
GPT's answer: 
` Tether (USDT) is a type of stablecoin originally issued on the Bitcoin blockchain via the Omni Layer protocol. Therefore, the stolen USDT was moved on the Bitcoin blockchain but wasn't "converted" to Bitcoin. The reason you might hear that the funds were "transferred into the Bitcoin chain" is because of this underlying architecture. They were always on the Bitcoin chain; it's just the token (USDT) that represents a different value than Bitcoin itself.`
I cannot confirm onchain this information, but it looks meaningful, since it Tether's official announcement